src/download/linux.md

```diff
@@ -42,6 +42,22 @@ apk add prismlauncher
 
 <div class="infobox top">
 
+# <img src="https://cdn.freebiesupply.com/logos/large/2x/altlinux-logo-black-and-white.png" height="20"> ALT Linux / <img src="https://static-00.iconduck.com/assets.00/aosc-icon-512x512-tv7gwhzl.png" height="20"> AOSC
+
+RPM Packages of ALT Linux on versions [sisyphus](https://packages.altlinux.org/en/sisyphus/srpms/prismlauncher/) and [p10](https://packages.altlinux.org/en/p10/srpms/prismlauncher/) are available for multiple architectures. [Prism Launcher](https://packages.aosc.io/packages/prismlauncher) is also available on AOSC.
+
+```bash
+sudo apt install prismlauncher
+# or
+su -c "apt install prismlauncher"
+```
+
+*Note: Prism Launcher in both distributions appears to be unmaintained, with AOSC still on version 6.1 and both versions of ALT Linux on version 6.3, so use packages at your own risk. If you are a user of these distributions, it is recommended that you either install the [flatpak](https://flathub.org/apps/details/org.prismlauncher.PrismLauncher) or [build the binary](/wiki/development/build-instructions/#linux) yourself.*
```

Comment:
this makes me a little unsure of adding this to the list honestly. most packages here are packaged by current and/or former maintainers, as well as known community members who keep the packages fairly up to date. using a package at 6.1 especially would be a poor recommendation since it is affected by multiple [known](https://github.com/PrismLauncher/PrismLauncher/security/advisories/GHSA-wxgx-8v36-mj2m) [vulnerabilities](https://github.com/PrismLauncher/PrismLauncher/security/advisories/GHSA-gq28-qx55-mh2r) that have been fixed for a while now. the flatpak/manual build suggestions are good though, but i would argue if we're recommending those to be used instead of these packages, there might not be much of a point in having them listed here in the first place - especially when it could put users at risk.

Reply:
alright, i might add an additional warning, however I'll probably leave them there just so that people know that they are in the official repo